packages/nodes-base/nodes/Postgres/v2/actions/database/update.operation.ts

```diff
@@ -82,7 +82,7 @@ const properties: INodeProperties[] = [
 			loadOptionsDependsOn: ['schema.value', 'table.value'],
 		},
 		default: '',
-		hint: 'The column that identifies the row(s) to modify',
+		hint: 'The column to use when matching rows in Google Sheets to the input items of this node. Usually an ID.',
```

Comment:
This should not be updated, we should be able to add a hint on the sheet node parameter

Reply:
as we updating hint for newer versions, we should update this for old versions as well

Reply:
this hint is in Postgres node, we should add it to the Google Sheet node

Reply:
got it, mb, updated service name to Postgres